client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "corona-tracker",
-  "version": "0.3.38",
+  "version": "0.3.39",
```

Comment:
Needs a version bump since I just merged in another change :)

Reply:
Done!